python/interpret_text/experimental/unified_information/unified_information_explainer.py

```diff
@@ -125,16 +128,16 @@ def explain_local(self, text, classes=None, predicted_label=None, true_label=Non
         self._optimize(num_iteration, lr=0.01, show_progress=True)
         local_importance_values = self._get_sigma()
         self.local_importance_values = local_importance_values
+        # predicted_label = self._wrapped_model.predict([X])
         return _create_local_explanation(
             classification=True,
             text_explanation=True,
             local_importance_values=np.array(local_importance_values)[1:-1],
             method="neural network",
             model_task="classification",
             features=self.parsed_sentence[1:-1],
-            classes=classes,
-            predicted_label=predicted_label,
-            true_label=true_label,
+            classes=self.classes,
+            true_label=y,
```

Comment:
why did we decide to remove the predicted label? it seems like it was useful information to have on the text explanation?

Reply:
That is the commented out code above. predicted_labels should be created from the model or predict_fn. not passed into explain_local. Similar to how introspective explainer does it. For now I dropped support for consistency. We can build it back up after we formalize predictions. I used the WrappedPytorchModel.predict there, but have not tested the e2e yet(hence it is commented out)